platform/src/main/java/org/stellar/anchor/platform/payment/observer/stellar/PaymentObservingAccountsManager.java

```diff
@@ -0,0 +1,157 @@
+package org.stellar.anchor.platform.payment.observer.stellar;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+import static java.time.temporal.ChronoUnit.HOURS;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.PostConstruct;
+import lombok.AllArgsConstructor;
+import org.stellar.anchor.api.exception.ValueValidationException;
+import org.stellar.anchor.platform.data.PaymentObservingAccount;
+import org.stellar.anchor.util.Log;
+
+public class PaymentObservingAccountsManager {
+  Map<String, ObservingAccount> allAccounts;
+  private final PaymentObservingAccountStore store;
+
+  public PaymentObservingAccountsManager(PaymentObservingAccountStore store) {
+    this.store = store;
+    allAccounts = new ConcurrentHashMap<>();
+  }
+
+  @PostConstruct
+  public void initialize() throws ValueValidationException {
+    List<PaymentObservingAccount> accounts = store.list();
+    for (PaymentObservingAccount account : accounts) {
+      ObservingAccount oa =
+          new ObservingAccount(account.getAccount(), account.getLastObserved(), true);
+      upsert(oa);
+    }
+
+    // Start the eviction task
+    ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+    scheduler.scheduleAtFixedRate(
+        this::evictAndPersist, 60, getEvictPeriod().getSeconds(), TimeUnit.SECONDS);
```

Comment:
I think we do need to make this configurable and use bigger default expiration times. Maybe 1 MONTH instead of 1 DAY?

Reply:
Yes, the function `getEvictPeriod()` is the place-holder for external configuration. [I plan to use ISO-8601 duration representation string notation used by `Duration.parse()`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html#parse(java.lang.CharSequence)). 

Does that make sense to you?

Reply:
Yes, it does!